apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -3,12 +3,22 @@ describe('When: I use the reading list feature', () => {
     cy.startAt('/');
   });
 
-  it('Then: I should see my reading list', () => {
+  xit('Then: I should see my reading list', () => {
     cy.get('[data-testing="toggle-reading-list"]').click();
 
     cy.get('[data-testing="reading-list-container"]').should(
       'contain.text',
       'My Reading List'
     );
   });
+
+  it('Test the snackbar message', function() {
```

Comment:
prefer to keep arrow functions for consistency

Reply:
sure. updated with arrow function